### DISCUSSION TITLE
Affinity Color Palette format support (.afpalette)?

### DISCUSSION THREAD

**Discussion by michaelokraj** [rsayfutdinov/Pikka#25] [Ideas]
Just got Pikka Pro and really like its versatility and possibilities. One thing is missing though: the support for Affinity format. Any chance to get that in the Pro version f.e.?

**Comment by rsayfutdinov** [comment]
> Just got Pikka Pro and really like its versatility and possibilities. One thing is missing though: the support for Affinity format. Any chance to get that in the Pro version f.e.?

Hi Michael,

I'm really glad you liked Pikka :)

I will take this into my list. Unfortunately, the quick google search not gave me any results about the file format. If you could share some .afpalette files with me, it would be a huge help.

Regards,
Ruslan

**Reply by michaelokraj** [reply]
Thank you for that quick reply.

Here is a Affinity page about their panel / palettes:
https://affinity.help/designer/en-US.lproj/index.html?page=pages/Panels/swatchesPanel.html?title=Swatches%20panel

Also feel free to check the affintiy forum to get to the "top"-contacts there – they are very resonsive:
https://forum.affinity.serif.com

Here are two examples of swatches in a Zip-File: https://www.dropbox.com/s/z0l5thze4f6aom6/Examples-Native-Format.zip?dl=0

Btw: to be transparent: Affinity supports the workaround with ".clr" files which can be imported. Nevertheless, as the Affinity community is growing, to see the native format supported can draw some attention to tools like Pikka ;)

**Reply by rsayfutdinov** [reply]
According to https://forum.affinity.serif.com/index.php?/topic/88691-afpalette-format-various-issues/
 .afpalette file format specification is not publicly available.

So I can't promise that Pikka will get the support for affinity palettes, but I will definitely look deeper into it.

Could you try this: Drag a color from Pikka and drop it into the Affinity Swatches Panel? Does it work?

**Reply by michaelokraj** [reply]
Understand. Yes, I read that somewhere, that it is not open to the public – but you are on the "other" side so I thought they share their insights to other developers. 

I tried drag and drop. But that does not work when I drag it 
- into the canvas
- into the palette or
- with alt-key + drag on the dock + app icon (this works with the affinity's palette format

If Pikka's drag-fomat is .clr, then one can only import it via the "import palette" in the color panel. Which is fine. With our testing, you can add Affinity as a supported app (with that workflow) in your documentation. 

Just another idea: Maybe this would be a nice workaround as well:
when you drag a color from the library to the finder / desktop, a .clr palette is generated. Then one can import that to affinity / apple / etc? Now you can't drag it to the desktop. What do you think?
